tests/dist/cpp/rpc_base.h

```diff
@@ -0,0 +1,29 @@
+#ifndef DIST_TEST_RPC_BASE_H_
+#define DIST_TEST_RPC_BASE_H_
+
+#include <iostream>
+#include <string>
+
+#include "../../../src/rpc/rpc_msg.h"
+#include "../../../src/rpc/tensorpipe/tp_communicator.h"
+#include "../../cpp/common.h"
+
+namespace {
+const int kNumSender = 30;
+const int kNumReceiver = 10;
+const int kNumMessage = 1024;
+const int kPort = 50090;
+const int kSizeTensor = 10 * 1024;
+const int kNumTensor = 30;
```

Comment:
Can we expose them to program arguments? This will make it easier to test different settings.

Reply:
DMLC provides utilities to parse key-value args. See https://github.com/dmlc/dmlc-core/blob/main/example/parameter.cc . Also JSON support: https://github.com/dmlc/dmlc-core/blob/main/include/dmlc/json.h

Reply:
I think more about it. As these configurations should be same both for clients and servers. That's why I configure them in the common head file. If we expose them to program arguments, we cannot control or assure these configurations are same for clients and servers. As these are specific for tests on specific machine, it's acceptable to configure like this instead of exposure?